.travis.yml

```diff
@@ -19,6 +19,10 @@ before_install:
 install:
   - make -C src/api bootstrap
   - make -C src/ui install
+  - |
+    if [[ -n "$TRAVIS_TAG" ]]; then
```

Comment:
As an aside, if you want to git rid of the inline bash from travis, then you could replace the `VERSION ?= dev` assignment in the Makefile with.

```Makefile
ifeq "$(VERSION)" ""
	override VERSION = dev
endif
```